monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py

```diff
@@ -12,9 +15,17 @@
 logger = logging.getLogger(__name__)
 
 DEFAULT_DIRS = ["/.ssh/", "/"]
+SSH_CREDENTIAL_COLLECTOR_TAG = "SSHCredentialsStolen"
+T1003_TAG = "T1003"
+T1005_TAG = "T1005"
+T1145_TAG = "T1145"
 
+SSH_COLLECTOR_EVENT_TAG = {SSH_CREDENTIAL_COLLECTOR_TAG, T1003_TAG, T1005_TAG, T1145_TAG}
```

Comment:
```suggestion
T1003_ATTACK_TECHNIQUE_TAG = "T1003"
T1005_ATTACK_TECHNIQUE_TAG = "T1005"
T1145_ATTACK_TECHNIQUE_TAG = "T1145"

SSH_COLLECTOR_EVENT_TAG = {SSH_CREDENTIAL_COLLECTOR_TAG, T1003_ATTACK_TECHNIQUE_TAG, T1005_ATTACK_TECHNIQUE_TAG, T1145_ATTACK_TECHNIQUE_TAG}
```

Reply:
Name attack tags all lowercase and prefix with "attack". Example: "attack-t1003"